drivers/clock_control/clock_control_mcux_ccm.c

```diff
@@ -170,17 +170,17 @@ static int mcux_ccm_get_subsys_rate(const struct device *dev,
 
 #ifdef CONFIG_I2S_MCUX_SAI
 	case IMX_CCM_SAI1_CLK:
-		*rate = CLOCK_GetFreq(kCLOCK_AudioPllClk) / 8
+		*rate = CLOCK_GetFreq(kCLOCK_AudioPllClk)
```

Comment:
From the RM: 
<img width="1125" alt="image" src="https://user-images.githubusercontent.com/7470161/163882349-239470b5-0168-4095-95b5-8935c0571e07.png">

The Audio PLL goes directly through 2 dividers, but no automatic DIV/8 is needed. This is also backed up by example driver code from the MCU Expresso SDK Builder.

Reply:
the default value is /4 and /2 which is /8, and currently there is no where that we change this vaule